phpBB/includes/acp/acp_inactive.php

```diff
@@ -211,20 +211,17 @@ function main($id, $mode)
 							 */
 							foreach ($messenger_collection_iterator as $messenger_method)
```

Comment:
Same here, the module description says `...remind (by sending an email)...`, so may be only use email method here. Otherwise migrate to notifications manager and change the description accordingly.